clients/nexus-iq/src/main/kotlin/NexusIqService.kt

```diff
@@ -102,7 +114,16 @@ interface NexusIqService {
         val reference: String,
         val severity: Float,
         val url: URI?
-    )
+    ) {
+        /**
+         * Return an identifier for the scoring system used for this issue. According to the documentation, the
+         * prefix of the reference determines the scoring system. See
+         * https://guides.sonatype.com/iqserver/technical-guides/sonatype-vuln-data/#how-is-a-vulnerability-score-severity-calculated
+         */
+        fun scoringSystem(): String =
+            if (reference.startsWith(PREFIX_SONATYPE)) SCORE_CVSS3
+            else SCORE_CVSS2
```

Comment:
Nit: We usually format simple `if` / `else` like this as a oneliner as a substitute for a ternary operator in Kotlin.